### ISSUE TITLE
crashes after listening to the second song

### ISSUE THREAD

**Issue by Venipa** [ytmdesktop/ytmdesktop#197]
```
[Window Title]
Error

[Main Instruction]
A JavaScript error occurred in the main process

[Content]
Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from undefined
    at C:\Users\Venipa\AppData\Local\Programs\youtube-music-desktop-app\resources\app.asar\providers\trayProvider.js:69:31
    at ClientRequest.<anonymous> (C:\Users\Venipa\AppData\Local\Programs\youtube-music-desktop-app\resources\app.asar\node_modules\base64-img\base64-img.js:81:21)
    at ClientRequest.emit (events.js:203:13)
    at Socket.socketErrorListener (_http_client.js:399:9)
    at Socket.emit (events.js:203:13)
    at emitErrorNT (internal/streams/destroy.js:91:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
    at processTicksAndRejections (internal/process/task_queues.js:77:11)

[OK]
```

**Comment by adlerluiz** [comment]
v1.9?

**Comment by Venipa** [comment]
yes 1.9.0, windows 10 (2004, b 19041.329)

**Comment by adlerluiz** [comment]
this was fixed in version 1.10
